internal/provider/resource_dashboard.go

```diff
@@ -81,6 +84,24 @@ func (r *DashboardResource) Schema(_ context.Context, _ resource.SchemaRequest,
 				Computed:    true,
 				CustomType:  jsontypes.NormalizedType{},
 			},
+			"timeframe": schema.StringAttribute{
+				Description: "The timeframe of the dashboard. It should be one of the following: last1hour, last12hours, last24hours, last7days, last30days, thisMonth, thisQuarter, thisYear, lastMonth, lastQuarter, lastYear",
+				Optional:    true,
+				Computed:    true,
```

Comment:
Why is timeframe marked as Computed?

Reply:
Its to handle null value as its marked as optional.

Reply:
> Its to handle null value as its marked as optional.

Ah, ta, so what value will it compute when null?

Reply:
just an empty string `""`. Because terraform store states based on response it expect some value than null